packages/host/app/components/matrix/room.gts

```diff
@@ -360,8 +367,8 @@ export default class Room extends Component<RoomArgs> {
   });
 
   private cleanup = () => {
-    if (this.subscribedRoom) {
-      this.cardService.unsubscribe(this.subscribedRoom, this.onCardChange);
+    if (this.subscribedRoom && this.api) {
+      this.api.unsubscribeFromChanges(this.subscribedRoom, this.onCardChange);
```

Comment:
this needs to be synchronous since it's in the destroy of the component, hence the need to save the `this.api` that we used earlier